drivers/shared/src/vic_log.c

```diff
@@ -29,80 +29,81 @@
  *****************************************************************************/
 
 #include <time.h>
-
 #include <vic_def.h>
 #include <vic_run.h>
 #include <vic_driver_shared.h>
 
 /******************************************************************************
- * @brief    Get string of current date and time.  Format YYMMDD-SSSSS.
+ * @brief    Finalize logging
  *****************************************************************************/
-char*
-get_current_datetime()
+void
+finalize_logging(void)
 {
-    const int size = 20;
-    char     *cdt = (char*)malloc(sizeof(char) * size);
-    char      ymd[8];
-    unsigned  seconds_since_midnight;
+    extern FILE *LOG_DEST;
 
-    if (cdt == NULL) {
-        return NULL;
+    if (!(LOG_DEST == stdout || LOG_DEST == stderr)) {
+        fclose(LOG_DEST);
```

Comment:
That's a good idea. This function should be called after all logging is completed, but resetting it to `stderr` would be good, so within the brackets, immediately after `fclose(LOG_DEST);` add `LOG_DEST = stderr;`. I'll amend the pull request.